synapseclient/client.py

```diff
@@ -808,29 +884,36 @@ def _findPrincipals(self, query_string: str) -> typing.List[UserGroupHeader]:
               u'lastName': u'Test',
               u'ownerId': u'1560002'},
              {u'displayName': ... }]
-
         """
         uri = "/userGroupHeaders?prefix=%s" % urllib_urlparse.quote(query_string)
         return [UserGroupHeader(**result) for result in self._GET_paginated(uri)]
 
-    def _get_certified_passing_record(self, userid: int) -> dict:
-        """Retrieve the Passing Record on the User Certification test for the given user.
+    def _get_certified_passing_record(
+        self, userid: int
+    ) -> Dict[str, Union[str, int, bool, list]]:
+        """
+        Retrieve the Passing Record on the User Certification test for the given user.
 
-        :params userid: Synapse user Id
+        Arguments:
+            userid: Synapse user Id
 
-        :returns: Synapse Passing Record
+        Returns:
+            Synapse Passing Record. A record of whether a given user passed a given test.
             https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/quiz/PassingRecord.html
```

Comment:
Could you update any links you find that are not in:
```
[text for link](link here)
```

To be in this format:
```
<link here>
```

By including the `<>` symbols it will render as a hyperlink on the docs pages.